lib/fs.js

```diff
@@ -363,6 +363,7 @@ function checkAborted(signal, callback) {
  *   encoding?: string | null;
  *   flag?: string;
  *   signal?: AbortSignal;
+ *   kPerRead?: Uint32;
```

Comment:
```suggestion
 *   kPerRead?: number;
```
JSDoc does not support different types of numbers/integers, only the primitive type of `number`.